### ISSUE TITLE
NaN calculating distance for the same point

### ISSUE THREAD

**Issue by ponzagiorgio** [xni06/JCoord#5]
If i try to calculate distance between 2 points having the same coordinates, sometimes i get a NaN. This is because the Math.acos is calculated on a number over than 1.
This happens for example for these coords (lat/lng)
- 45.055094,7.647369
- 45.091711,7.661622

My test
double myLat = 45.091711;//45.055094;
double myLng = 7.661622;//7.647369;
LatLng origin = new LatLng(myLat, myLng);
LatLng dest = new LatLng(myLat, myLng);
System.out.println(origin.distance(dest));

My suggestions:
1) do some check and return 0 if coords are same, without calculating the formula OR
2) at the end of the 'distance' function, write

return Double.isFinite(d) ? d : 0D;

**Comment by xni06** [comment]
Makes sense. Create a pull request including the change and unit test and we'll slip it in. Thanks

**Comment by ponzagiorgio** [comment]
I am absolutely new to github, please give me some time to understand how to use it. I'll do the pull request for sure. Thanks again
